### ISSUE TITLE
LVDS -> MIPI CSI

### ISSUE THREAD

**Issue by sdrobertw** [96boards/mezzanine-community#77]
Any thoughts on this one? 

@mwelling @ric96

**Comment by mwelling** [comment]
So LVDS camera/display output converted to CSI before going to the high speed header?

Might be able to leverage something like this:
http://www.analog.com/media/en/technical-documentation/data-sheets/ADV7782.pdf

What are you trying to accomplish?

**Comment by sdrobertw** [comment]
@mwelling Yeah, Joyce (96Boards PM) shared some Lontium and Lattice chips that would do the trick too, I guess...

I don't know exact reasons, I will look into it. This mezzanine idea was brought up by Yang. Thought I would at least throw it up here as a place holder in case there are any discussions around it.

**Comment by Yang-96Boards** [comment]
This is to take 4 x 1080p GMSL or LVDS cameras mux into 1 CSI2 - automotive vision subsystem use case.

**Comment by mwelling** [comment]
If there is a use case then I think it is certainly worthy. Thanks for posting.

**Comment by Yang-96Boards** [comment]
Thanks Michael - if anyone got one of those I ll personally be buying a
dozen right away :-P... we really need this.
